file2.json

```diff
@@ -0,0 +1 @@
+﻿[{"Name":"Maciej","Surname":"Dysput","Occupation":"Automatics and Robotics","Age":23,"Id":31},{"Name":"Robert","Surname":"Wojdick","Occupation":"Philosophy","Age":54,"Id":577}]
```

Comment:
This shouldn't be included.  Please add this to gitignore

Reply:
added